### DISCUSSION TITLE
JuiceFS 元数据库读写分离

### DISCUSSION THREAD

**Discussion by status-code-404** [juicedata/juicefs#5024] [Q&A] [answered]
JuiceFS在与元数据库交互时为保证数据的一致性 大部分select 都带了for update
对一些实时性并没有那么高的系统，或者只读节点来说，不带事务锁的select 只读就能满足业务要求的情况下
是否有相关手段和配置设置只读节点或者让一些select降级不带for update或事务的动作，使这些请求可以被打到从库上

**Comment by davies** [comment, marked as answer]
需要写入的挂载点，是不能使用从库来做读写分离的，因为 从库的 复制延迟会导致一致性问题。

只读挂载点，是可以使用从库来挂载的。

**Reply by status-code-404** [reply]
好的 谢谢大佬
